CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/TranslationConversationContributionProvider.java

```diff
@@ -248,6 +248,10 @@ public String toString() {
         }
     }
 
+    /**
+     * Plugin to set the translation.
+     */
+    @PluginInfo(pluginType = PluginType.UPDATE, tags = {"LOW LEVEL"})
     private class SetTranslationPlugin extends SimpleEditPlugin {
```

Comment:
Given the plugin appears to be setting the relevant transaction attribute, I would go with MODIFY rather than LOW LEVEL